pkg/actions/nodegroup/create.go

```diff
@@ -86,20 +76,27 @@ func (m *Manager) Create(options CreateOpts, nodegroupFilter filter.NodeGroupFil
 		return err
 	}
 
+	printer := printers.NewJSONPrinter()
 	if err := printer.LogObj(logger.Debug, "cfg.json = \\\n%s\n", cfg); err != nil {
 		return err
 	}
 
-	// TODO
-	if err := ctl.ValidateClusterForCompatibility(cfg, stackManager); err != nil {
-		return errors.Wrap(err, "cluster compatibility check failed")
+	if !isUnownedCluster {
```

Comment:
`isUnownedCluster` is only  used in an `if & else` statement or an `if !isUnownedCluster`, so could we called it `isOwnedCluster` instead so I never have to parse the ! in my head :smile: :sweat_smile: